test/test_unistd.rs

```diff
@@ -242,7 +242,7 @@ fn test_fpathconf_limited() {
 #[test]
 fn test_pathconf_limited() {
     // AFAIK, PATH_MAX is limited on all platforms, so it makes a good test
-    let path_max = pathconf(".", PathconfVar::PATH_MAX);
+    let path_max = pathconf("/", PathconfVar::PATH_MAX);
```

Comment:
See the commit message for that change. It was failing when the tests were run in parallel. Presumably one of them is doing something that makes . No longer exist.